### PR TITLE
More schema manager love

### PR DESCRIPTION
**Pull request by GawainLynch**

### CODE REVIEW

File: src/Provider/DatabaseSchemaProvider.php [hunk 1]
```diff
@@ -19,15 +19,18 @@ function ($app) {
...
+
             return new \Pimple([
                 // @codingStandardsIgnoreStart
-                'authtoken'  => $app->share(function () use ($app) { return new Table\AuthToken(); }),
-                'cron'       => $app->share(function () use ($app) { return new Table\Cron(); }),
-                'log_change' => $app->share(function () use ($app) { return new Table\LogChange(); }),
-                'log_system' => $app->share(function () use ($app) { return new Table\LogSystem(); }),
-                'relations'  => $app->share(function () use ($app) { return new Table\Relations(); }),
-                'taxonomy'   => $app->share(function () use ($app) { return new Table\Taxonomy(); }),
-                'users'      => $app->share(function () use ($app) { return new Table\Users(); }),
```
Comment (CarsonF) [inline src/Provider/DatabaseSchemaProvider.php]: Don't need to use $app here

Comment (GawainLynch) [inline src/Provider/DatabaseSchemaProvider.php]: Copy/pasta derp

File: src/Database/Table/ContentType.php [hunk 2]
```diff
@@ -42,4 +78,142 @@ protected function setPrimaryKey()
...
+    {
+        return isset($this->typeMap[$type]);
+    }
+
+    /**
+     * Add the contenttype's specific fields.
+     *
+     * @param string $fieldName
+     * @param string $type
+     * @parma boolean $addIndex
```
Comment (CarsonF) [inline src/Database/Table/ContentType.php]: What's a `parma`? I don't think they have those outside of AU

Comment (rossriley) [inline src/Database/Table/ContentType.php]: Delicacy Injection:

Parma

Comment (GawainLynch) [inline src/Database/Table/ContentType.php]: Damn it, since when doesn't GitBookHub not have a "Like" button!

**Comment by bobdenotter** [comment]
Whoop!

**Pull request merged**